unix_commands/cat.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python3
```

Comment:
Эта строчка "скажет" скрипту использовать системный питон, но мы хотим использовать питон из текущего активного окружения (будь то conda или virtualenv). Понятное дело, что мы не можем прописать сюда все возможные варианты, но есть "магическая" строчка, которая сделает всю работу
```suggestion
#!/usr/bin/env python3
```